### PR TITLE
Feature: outline appearance

### PR DESCRIPTION
**Pull request by curryy**

### CODE REVIEW

**Comment by hzub** [comment]
NICE JOB! 👍 
There is one thing, through: IMO it's crucial to add proper stories/styles to "FormField" component as well. It is going to be one of the most used components in our projects, hence necessity of proper documentation.

**Comment by hzub** [comment]
There are minor display bugs on Chrome (`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.120 Safari/537.36`):
1) Up/down glyph on outline select is heavily displaced:
Zrzut ekranu 2019-10-28 o 07 37 31
2) Down arrow on outline tabs is not aligned properly:
Zrzut ekranu 2019-10-28 o 07 38 17

**Comment by hzub** [comment]
Please add easing on animating outline select menu, it's linear now

File: packages/select-menu/src/styles.tsx [hunk 1]
```diff
@@ -102,4 +84,77 @@ export const TextContainer = styled.div`
   justify-content: center;
   text-align: center;
 `
-export const StyledPopover = styled(Popover)``
+export const InputAsSelectButtonComponent = styled(TextInput)`
+  cursor: pointer;
```
Comment (hzub) [inline packages/select-menu/src/styles.tsx]: Selector control's height should match option's height, also transition between them should be seamless:
Zrzut ekranu 2019-10-30 o 11 51 01

**Comment by Idered** [comment]
I'm working on some fixes to this PR in a separate PR available here #62

**Pull request merged**